install.sh

```diff
@@ -31,42 +31,39 @@ tmp_dir="$tmp/install.sh.$$"
 # do_download URL DIR
 do_download() {
   echo "Downloading $1"
-  if [[ $(type -t wget) ]]; then
+  if [[ $(command -v wget) ]]; then
     wget -q -O "$2" "$1" >/dev/null
-  elif [[ $(type -t curl) ]]; then
+  elif [[ $(command -v curl) ]]; then
     curl --fail -sSL -o "$2" "$1" &>/dev/null || true
   else
     error "Could not find wget or curl"
     return 1
   fi
 }
 
-# get_latest_version URL
+# get_latest_version
 get_latest_version() {
-  versionloc=${tmp_dir}/summon.version
-  versionfile=$(do_download "${1}" "${versionloc}")
-  if [ -f "${versionloc}" ]; then
-    local version=$(cat ${versionloc} | grep -o -e "[[:digit:]].[[:digit:]]*.[[:digit:]]*")
-    echo "${version}"
+  if [[ $(command -v curl) ]]; then
+    curl --silent "https://api.github.com/repos/cyberark/summon/releases/latest" | # Get latest release from GitHub api
+    grep '"tag_name":' |                                            # Get tag line
+    sed -E 's/.*"([^"]+)".*/\1/'                                    # Pluck JSON value
+  else
+    error "Could not find curl"
+    return 1
   fi
 }
 
-LATEST_VERSION=$(get_latest_version 'https://raw.githubusercontent.com/cyberark/summon/master/pkg/summon/version.go')
-# TODO: This should be removed after we publish the newest version (v0.6.9+)
-if [[ -z "$LATEST_VERSION" ]]; then
-  echo "Trying the old version endpoint..."
-  LATEST_VERSION=$(get_latest_version 'https://raw.githubusercontent.com/cyberark/summon/master/version.go')
-fi
+LATEST_VERSION=$(get_latest_version)
 
 echo "Using version number: v$LATEST_VERSION"
 
 BASEURL="https://github.com/cyberark/summon/releases/download/"
-URL=${BASEURL}"v${LATEST_VERSION}/summon-${DISTRO}-amd64.tar.gz"
+URL=${BASEURL}"${LATEST_VERSION}/summon-${DISTRO}-amd64.tar.gz"
 
 ZIP_PATH="${tmp_dir}/summon.tar.gz"
 do_download ${URL} ${ZIP_PATH}
 
-echo "Installing summon v${LATEST_VERSION} into /usr/local/bin"
+echo "Installing summon ${LATEST_VERSION} into /usr/local/bin"
```

Comment:
I think when displaying versions we should leave the `v` prefix since it's standard for most other tools

Reply:
agreed and addressed :)